recipes-nodes/reth/reth.inc

```diff
@@ -24,4 +24,23 @@ do_install:append() {
     install -m 0755 ${THISDIR}/init ${D}${sysconfdir}/init.d/reth
 }
 TOOLCHAIN = "clang"
-RDEPENDS:${PN} += "disk-encryption config-parser date-sync reth-sync"
+RDEPENDS:${PN} += "disk-encryption config-parser"
+
+python () {
```

Comment:
Wouldn't `INCLUDE_RCLONE ?= 1` do the trick?

Reply:
Nope, because we also want the 3rd option we discussed today where the image builder to completely exclude rclone from the image and not include it at all.
This way here, it allows all 3 options we discussed today and also described in the PR description